generator/src/templates/assembly-compile-new.tsconfig.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Yes, but since we're not manipulating the theia/package.json, we don't have to. This PR is a first step that should allow folks to use the linkless build process in day-to-day development. Once we've changed the docker build scripts to use the new process, the init-sources files will go away.

Reply:
having two definition for the same stuff is always breaking. People only modify at a single place.
I would not reproduce the nightmare it was on plug-in registry when there was both all meta.yaml and vscode-extensions.json in parallel.

Also I would not opt-in for `-new` as there is no '-old' assembly, just that they're different